kerberos5/include/config.h

```diff
@@ -694,7 +696,9 @@
 #define HAVE_RES_NDESTROY 1
 
 /* Define to 1 if you have the `res_nsearch' function. */
+#if __RES >= 19991006
```

Comment:
cc @cschuber - is there any automation involved in the generation of config.h on krb5 updates?

Reply:
This likely needs a comment, but this doesn't feel like the right place to solve this...